Userland/Libraries/LibCore/SocketAddress.h

```diff
@@ -8,11 +8,18 @@
 #pragma once
 
 #include <AK/IPv4Address.h>
-#include <arpa/inet.h>
-#include <netinet/in.h>
 #include <string.h>
-#include <sys/socket.h>
-#include <sys/un.h>
+
+#ifndef AK_OS_WINDOWS
+#    include <arpa/inet.h>
+#    include <netinet/in.h>
+#    include <sys/socket.h>
+#    include <sys/un.h>
+#else
+#    include <WS2tcpip.h>
+#    include <WinSock2.h>
+#    include <afunix.h>
+#endif
```

Comment:
IMO, having `#if defined(AK_OS_WINDOWS) ... #else` would be nicer to read. Same for one or two other places.

Also, the mix of using `#ifdef/#ifndef` sometimes and `#if defined()` others is very slightly bothering me, but this is already a thing throughout the codebase so eh.

Reply:
Sure, I'll do a scrub to make sure each file is self-consistent. Might be worth a global scrub :yakbait: but I don't think that's in scope here yet :)